Chainsaw Updated sprites

BenjaminZuriel opened the pull request:
Updated the heavy attack, right swing and world sprites.

JMartinez9820 commented:
Hey Ben, I just checked your pull request and everything looks good. I noticed that the first frame of the back attack with green blood had its offsets misplaced and the chainsaw blade was lower than the others when I compared them. I have fixed the offsets and modified that sprite using an existing sprite so it matches with the other sprites. Feel free to check out the pull request I submitted to your branch and test it, thanks!

BenjaminZuriel commented:
> 
> 
> Hey Ben, I just checked your pull request and everything looks good. I noticed that the first frame of the back attack with green blood had its offsets misplaced and the chainsaw blade was lower than the others when I compared them. I have fixed the offsets and modified that sprite using an existing sprite so it matches with the other sprites. Feel free to check out the pull request I submitted to your branch and test it, thanks!

Thanks mate.
